diabetes_regression/training/train_aml.py

```diff
@@ -120,10 +123,11 @@ def main():
         train_args = {}
 
     # Log the training parameters
-    print(f"Parameters: {train_args}")
+    logging.info(f"Parameters: {train_args}")
```

Comment:
I wonder if we should log this as a custom event in App Insights, given that we have a dictionary (if I'm reading this correctly, if not, carry on) with all these args as context. @sundar-swetha  thoughts?

Reply:
Removed this log info since it's logging each args inside the loop.